src/redux/actions/authActions.ts

```diff
@@ -0,0 +1,63 @@
+import { ActionTypes } from "./actionTypes";
+import { IPayloadAction, createPayloadAction, createAction } from "./actionCreators";
+import { IAuth } from "../../models/applicationState";
+import { Dispatch, Action } from "redux";
+import { IpcRendererProxy } from "../../common/ipcRendererProxy";
+
+/**
+ * Actions which manage users auth
+ * @member signIn - Allows to sign in to the application
+ * @member signOut - Allows to sign out from the application
+ */
+export default interface IAuthActions {
+    signIn(accessToken: IAuth): Promise<void>;
+    signOut(): Promise<void>;
+}
+
+/**
+ * Sign in to the application
+ * @param accessToken - Auth to the application
+ */
+export function signIn(accessToken: string): (disptach: Dispatch) => Promise<void> {
+    return (dispatch: Dispatch) => {
+        return IpcRendererProxy.send("SIGN_IN", accessToken)
```

Comment:
Where does this `SIGN_IN` come from? How is it used ? Does the framework add the suffix _SUCCESS ?

Reply:
It's a thing to the electron.js (desktop app). We could bind it ealier like this:
`ipcMainProxy.register("SIGN_IN", onSignIn);`